localization/gyro_odometer/README.md

```diff
@@ -28,4 +27,12 @@
 
 ## Assumptions / Known limits
 
-TBD.
+- [Assumption] The frame_id of input twist message must be base_link.
```

Comment:
This already can make sense but  `The frame_id of input twist message must be set to base_link` is more fluent, I suppose.

Reply:
Thank you!
fixed https://github.com/autowarefoundation/autoware.universe/pull/412/commits/9050b3d6e204e1c6657fe091ba2e067d71783c77